packages/dashboard-core-plugins/src/linker/LinkerOverlayContent.tsx

```diff
@@ -52,20 +70,55 @@ export class LinkerOverlayContent extends Component<
     disabled: 'false',
   };
 
+  static getLabelForFilter(
+    columnType: string,
+    filterType: FilterTypeValue
+  ): string {
+    try {
+      if (
+        TableUtils.isNumberType(columnType) ||
+        TableUtils.isCharType(columnType)
+      ) {
+        return getLabelForNumberFilter(filterType);
+      }
+      if (TableUtils.isTextType(columnType)) {
+        return getLabelForTextFilter(filterType);
+      }
+      if (TableUtils.isDateType(columnType)) {
+        return getLabelForDateFilter(filterType);
+      }
+      if (TableUtils.isBooleanType(columnType)) {
+        return getLabelForBooleanFilter(filterType);
+      }
+      throw new Error(`Unrecognized column type: ${columnType}`);
+    } catch (e) {
+      log.warn(e);
+      return '';
+    }
+  }
+
   constructor(props: LinkerOverlayContentProps) {
     super(props);
 
     this.handleMouseMove = this.handleMouseMove.bind(this);
+    this.handleKeyDown = this.handleKeyDown.bind(this);
+    this.handleKeyUp = this.handleKeyUp.bind(this);
     this.handleEscapePressed = this.handleEscapePressed.bind(this);
+    this.handleComparisonOperatorChanged = this.handleComparisonOperatorChanged.bind(
+      this
+    );
 
     this.state = {
       mouseX: undefined,
       mouseY: undefined,
+      isAltPressed: false,
```

Comment:
Instead of having an `isAltPressed` state variable, make the state variable either something like `isDeleting: boolean`, or even `mode: 'select' | 'delete'` or something to that effect. We're using Alt to trigger delete mode here, but it wouldn't be unusual to trigger it via some other method.